coderd/provisionerdserver/provisionerdserver.go

```diff
@@ -1098,7 +1098,8 @@ func (s *server) FailJob(ctx context.Context, failJob *proto.FailedJob) (*proto.
 func (s *server) notifyWorkspaceBuildFailed(ctx context.Context, workspace database.Workspace, build database.WorkspaceBuild) {
 	var reason string
 	if build.Reason.Valid() && build.Reason == database.BuildReasonInitiator {
-		return // failed workspace build initiated by a user should not notify
+		s.notifyWorkspaceManualBuildFailed(ctx, workspace, build)
```

Comment:
Where do we specify that `build.Reason == database.BuildReasonInitiator` defines a manual build?

Reply:
I think this is on the very high level in the API: [link](https://github.com/coder/coder/blob/ded612d3ecfe75fb22a05fafead414551b0ad396/coderd/workspaces.go#L650), unless I misread it?

Reply:
Cool, thanks 👍 seems like https://github.com/coder/coder/blob/dcab6fa5a4de57a899af2257b1a29508e9f37dfb/codersdk/workspacebuilds.go#L39-L42 describes it clearly as well.